source/deprecations/v2.x.html.md

```diff
@@ -615,3 +615,45 @@ export default Router.map(function() {
   });
 });
 ```
+
+#### Enumerable.contains
+
+##### until: 3.0.0
+##### id: ember-runtime.enumerable-contains
+
+The `Enumerable.contains` and `Array.contains` methods were deprecated in favor of `Enumerable.includes` and `Array.includes` 
```

Comment:
Can we represent theses as `Enumerable#contains`? I tend to treat `Enumerable.contains` as a method that is literally on that thing (class usually), and `#contains` as a method that exists on an instance of that thing (object, mixin applied to object).